transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -428,4 +437,6 @@ private static native DatagramSocketAddress recvFromAddress(
     private static native void setSoLinger(int fd, int soLinger) throws IOException;
     private static native void setTcpDeferAccept(int fd, int deferAccept) throws IOException;
     private static native void setTcpQuickAck(int fd, int quickAck) throws IOException;
+
+    private static native PeerCredentials peerCredentials(int fd) throws IOException;
```

Comment:
Moved it up.  I agree that is the convention but if you look up a bit I was asked to remove the get prefix ....